src/libs/ascent/runtimes/flow_filters/ascent_runtime_vtkh_filters.cpp

```diff
@@ -3794,17 +3805,40 @@ VTKHUniformGrid::execute()
     }
     std::shared_ptr<VTKHCollection> collection = data_object->as_vtkh_collection();
 
-    std::string field = params()["field"].as_string();
-    if(!collection->has_field(field))
+    //std::string field = params()["field"].as_string();
```

Comment:
I think we want to support both `field` and `fields`, since we have folks using the `field` form.
If only `field` is provided, we can add it to the string list.
If both are provided, we can just use the `fields` value.